lib/tasks/mastodon.rake

```diff
@@ -19,6 +19,36 @@ namespace :mastodon do
     puts "User #{email} confirmed."
   end
 
+  desc 'Add a user by providing their email, username and initial password.' \
+       'The user would receive a confirmation mail and then would be able to reset his password for logging in.'
```

Comment:
those grammar nits I left out:

active voice, will instead of would

> The user will receive a confirmation email, then must reset their password before logging in.